yarn.lock

```diff
@@ -4033,7 +4033,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"eslint-import-resolver-typescript@npm:^3.5.2":
+"eslint-import-resolver-typescript@npm:3.5.3, eslint-import-resolver-typescript@npm:^3.5.2":
```

Comment:
This is strange that the package was there in another version, yet without adding it to `package.json` we had problems like this:
![image](https://user-images.githubusercontent.com/66744401/222749841-2e28f1b8-50af-4c4b-bbaf-743e4b5d0869.png)
![image](https://user-images.githubusercontent.com/66744401/222749896-e8df34a0-9680-4f85-b324-c970fc04009d.png)

Reply:
That's strange, really